staging/src/k8s.io/dynamic-resource-allocation/structured/pools.go

```diff
@@ -21,36 +21,32 @@ import (
 	"fmt"
 
 	v1 "k8s.io/api/core/v1"
-	resourceapi "k8s.io/api/resource/v1alpha3"
-	"k8s.io/apimachinery/pkg/labels"
-	resourcelisters "k8s.io/client-go/listers/resource/v1alpha3"
 	"k8s.io/component-helpers/scheduling/corev1/nodeaffinity"
+	"k8s.io/dynamic-resource-allocation/api"
```

Comment:
Can you declare this to be draapi or dapi? Seeing API makes me think this is coming from somewhere else than DRA.

Reply:
I was wondering about this. There is other code where "api" is used for "the API we work with" instead of choosing a longer name, but I agree that a longer name may help with readability. I'm okay with "draapi".